### DISCUSSION TITLE
License update

### DISCUSSION THREAD

**Discussion by UPfeifer** [rustdesk/rustdesk#7761] [Q&A]
Hello Rustdesk, we have purchased a license for 60 users and 200 devices for one year.
We would like to extend the license to 300 devices.
We are only offered a new license for 60 users and 300 devices.

Is this the only option?
Then we would have paid for the first license for 10 months for nothing.

**Comment by rustdesk** [comment]
You can upgrade license, here is doc: https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/license/#renewupgrade-license
